share/ansible_navigator/utils/catalog_collections.py

```diff
@@ -91,7 +93,12 @@ def _catalog_plugins(self, collection: Dict) -> None:
 
     @staticmethod
     def _generate_chksum(file_path: str, relative_path: str) -> Dict:
-        """generate a std checksum for a file"""
+        """Generate a standard checksum for a file.
+
+        :param file_path: The path to the file to generate a checksum for
+        :param relative_path: The relative path within the collection directory structure
+        :returns: A diction or details about the file, including the checksum
```

Comment:
"diction or details"? Feels like some autocorrect gone rogue. "Of"?

Reply:
lol